adapters/handlers/rest/middlewares.go

```diff
@@ -59,6 +62,25 @@ func addHandleRoot(next http.Handler) http.Handler {
 	})
 }
 
+// metaSchemaRegexp is used to intercept requests and redirect them to a dedicated http server independent of swagger
+var metaSchemaRegexp = regexp.MustCompile("/raft/*")
+
+// addMetaSchemaMiddleware will inject a middleware that will catch all requests matching metaSchemaRegexp.
+// If the request match, it will route it to a dedicated http.Handler and skip the next middleware.
+// If the request doesn't match, it will continue to the next handler.
+func addMetaSchemaMiddleware(next http.Handler, metaSchemaHandler *schema.Handler) http.Handler {
```

Comment:
I would change the name, having `meta` in the name for me makes it think of a `/v1/meta` endpoint `middleware` which it's not what it does.
Can we rename to `addNativeHTTPMiddleware` for example?